common/src/main/java/com/wynntils/models/items/encoding/type/EncodingSettings.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.encoding.type;
+
+/**
+ * Settings for encoding items.
+ * @param extendedIdentificationEncoding Whether to use extended identification encoding. This only affects identifiable items.
+ * @param shareItemName Whether to share the item name between the item name and the item lore. This only affects crafted and custom items.
```

Comment:
"share the item name between the item name and the item lore". Sounds weird, not sure what is meant.